lib/mappings/bib-mapping.json

```diff
@@ -184,25 +184,17 @@
 
     ]
   },
-  "contributorLiteralByRole": {
+  "nameTitleRole": {
```

Comment:
Do we need to define this in index-schema.js?

Reply:
oh sorry! this work is not actually going to happen after all.